neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -122,6 +122,26 @@ public UInt160[] GetScriptHashesForVerifying(Snapshot snapshot)
             return hashes.OrderBy(p => p).ToArray();
         }
 
+        public virtual bool Reverify(Snapshot snapshot, IEnumerable<Transaction> mempool)
```

Comment:
@igormcoelho, `Reverify` is still not checking the `GAS` spent during application.

Reply:
This will be a future discussion brother... right now focus is just basic tx ops.

Reply:
ok, broda.